aws_topology/stackstate_checks/aws_topology/resources/elb_classic.py

```diff
@@ -13,78 +42,105 @@ class ELBClassicCollector(RegisteredResourceCollector):
     COMPONENT_TYPE = "aws.elb_classic"
     CLOUDFORMATION_TYPE = "AWS::ElasticLoadBalancing::LoadBalancer"
 
+    @set_required_access_v2("elasticloadbalancing:DescribeTags")
+    def collect_tag_page(self, elb_names):
+        if len(elb_names) <= 20:
```

Comment:
Either add a comment here about what this 20 means (api limit) or create a class variable so it's a bit more clear